Bio/SeqIO/IgIO.py

```diff
@@ -23,11 +21,11 @@
 class IgIterator(SequenceIterator):
     """Parser for IntelliGenetics files."""
 
-    def __init__(self, source, alphabet=single_letter_alphabet):
+    def __init__(self, source, alphabet=None):
```

Comment:
@mdehoon should we just remove alphabet argument when it is the final argument (here and cases like it)?

Reply:
Ywa

Reply:
OK - some done on #3126 but the rest of the files in this pull request need updating still.